README.md

```diff
@@ -30,6 +30,8 @@ sudo ln -s /usr/lib/llvm-15/bin/clang{,++,-format,-tidy} /usr/local/bin
 
 Python 3 is required to automatically build and link unit tests. Python 3 comes preinstalled on most Linux distributions, and is available as a package on all others.
 
+To build the doxygen documentation, doxygen and graphviz are required.
```

Comment:
Include the actual command/modify the list of commands that we use.

Reply:
Yeah, that was for your benefit, I'm thinking of adding a fleshed out section detailing how to build and view them and also pointing the user towards the gh pages hosted version.

Reply:
Those would both be good changes to make

Reply:
Done